app/src/main/res/values/strings.xml

```diff
@@ -42,4 +44,25 @@
     </string>
     <string name="sync_internet_stop_confirm_message">The sync hasn’t finished yet.</string>
 
+    <!-- Onboarding -->
+    <string name="onboarding_title_1">Welcome\nto Relaynet!</string>
```

Comment:
Would the container for these titles wrap to the second line if the text is too long? I'm asking because I think the final version shouldn't force line breaks and it should also support translations.

Reply:
This line breaks are only to make the text look good yes. But the alternative would be an awkward title width in the view, to force this breaks to happen. This way, each translation could have its own line breaks, or none, to ensure it looks good.

Reply:
To be honest, I think it looks weird to break sentences that would easily fit in one line (the first two titles for sure; maybe the 3rd too). The title in the first screen looks especially weird to me because its width is inconsistent with the rest and the second line is longer than the first time.

Sorry I didn't flag it on Figma! I didn't realise it was intentional.